mono/sgen/sgen-conf.h

```diff
@@ -22,7 +22,14 @@ typedef guint32 mword;
 typedef guint64 mword;
 #endif
 
-typedef mword SgenDescriptor;
+#if TARGET_SIZEOF_VOID_P == 4
```

Comment:
I would ask for a typedef somewhere of `target_gsize`, however that won't work with the single configure/compile/link goal.